### PR TITLE
Fix component invocation when using nested / namespaced components

### PR DESCRIPTION
**Pull request by NullVoxPopuli**

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
I'm working on finding an HTML validator for the tests atm. Testing against snapshots only assures accidental changes don't make it in

File: tests/index.test.js [hunk 1]
```diff
@@ -3,12 +3,22 @@ const unified = require('unified');
...
+  test(`${filePath}: it should keep all the HBS and HTML`, () => {
+    const file = stack.processSync(fs.readFileSync(__dirname + filePath));
+
+    const contents = file.toString();
+
+    expect(contents).toMatchSnapshot();
+
+    // Validate that the contents are parseable
+    // this will throw an exception if they are not
+    parse(contents);
```
Comment (NullVoxPopuli) [inline tests/index.test.js:22]: this currently causes the test to fail :)

Comment (josemarluedke) [reply tests/index.test.js:22]: :+1:

**Comment by josemarluedke** [comment]
@NullVoxPopuli Thank you for the reproduction.

I just pushed in your branch with a fix.

**Comment by josemarluedke** [comment]
Published in v0.3.2.

**Comment by NullVoxPopuli** [comment]
oof, I need to figure out how not to leak that :thinking:

**Comment by NullVoxPopuli** [comment]
I have a second one incoming :)

**Pull request merged**